src/trusted/desc/win/nacl_desc.c

```diff
@@ -40,16 +40,9 @@ int32_t NaClAbiStatHostDescStatXlateCtor(struct nacl_abi_stat   *dst,
     return -NACL_ABI_EOVERFLOW;
   }
 
-  dst->nacl_abi_st_dev = 0;
-#if defined(NACL_MASK_INODES)
```

Comment:
Why are we dropping this?

Reply:
This was designed in NaCl to mask host inodes on stats. Since we're now returning our own "faked" rustposix inodes, it's fine to return them to the user.